torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -1745,3 +1752,19 @@ def rref_tensor_is_owner(rref_var):
 
         res = rref_tensor_is_owner(rref_var)
         self.assertEqual(res, False)
+
+    @dist_init
+    def test_local_rref_with_script_module(self):
+        # create a local RRef that hold a ScriptModule
+        script_mod = MyScriptModule()
+        rref_module = rpc.RRef(script_mod._c)
```

Comment:
How about using rpc_sync call here?  rref_module = rpc_sync(workName, create_local_rref, args=())

def create_local_rref:
     return rpc.RRef(script_mod._c)

Reply:
what's the reason we are wrapping this with `rpc_sync` call?

Reply:
I think this is the use case we want? otherwise how the rref_module is passed?

rref_module = rpc_sync(workName, create_local_rref, args=())
rpc_sync(workName, run_rref_forward, args=(rref_module, ...))

Reply:
sounds good I will add this use case.